CONTRIBUTING.md

```diff
@@ -107,13 +105,13 @@ test('should work', async () => {
 })
 ```
 
-Some common test helpers, e.g. `testDir`, `isBuild` or `editFile` are also available in the utils. Source code is located at `playground/test-utils.ts`.
+Some common test helpers (e.g., `testDir`, `isBuild`, or `editFile`) are also available in the utils. Source code is located at `playground/test-utils.ts`.
```

Comment:
The use of comma after `.e.g.` feels a bit weird to me, I think we should remove the comma. There are other places with this pattern too.

Reply:
Five of six writing style guides lean toward using a comma after `e.g.` ([source](https://english.stackexchange.com/a/16215/5128))

If there are other instances outside of this page that omit the comma, I think they should be updated as well. But if you feel strongly against, I'm fine with removing it.

Reply:
The source did mention that it's not common for a comma to appear in British English, which I'm familiar with 😄 But up until now this is probably my first time seeing a comma after `.e.g.` on the web. I still feel like we should remove it unless others have different opinions of it.